script.js

```diff
@@ -1,4 +1,100 @@
+//global variables
+var gameMode = "rollDice"; //3 modes, get number, combine number, find winner
+var diceNumbers = []; //for each set of 2 dice rolls
+var chosenNumbersInARound = []; //for all players
+var allChosenNumbers = [];
+// var playerScore = 0;
+var player1Score = 0;
+var player2Score = 0;
+var playerNumber = 1;
+var largerNumber = 0;
+
+//generate random number 1 to 6
+function rollDice() {
+  return Math.ceil(Math.random() * 6);
+}
+
+//roll 2 numbers
+function getDiceNumbers() {
```

Comment:
This function is handling both the rolling of the dice as well as generating a message. Try to make your functions only do 1 thing